scripts/package-build/linux-kernel/build.py

```
@@ -242,17 +244,17 @@ def build_mellanox_ofed():
 
 
 def build_realtek_r8152():
     """Build Realtek r8152"""
-    run(['./build-realtek-r8152.py'], check=True)
+    run(['./build-realtek-r8152.sh'], check=True)
 
 
 def build_jool():
     """Build Jool"""
-    run(['echo y | ./build-jool.py'], check=True, shell=True)
+    run(['echo y | ./build-jool.sh'], check=True, shell=True)
```

Ruff (0.16.1)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)